libs/vue/src/utils.ts

```diff
@@ -61,3 +62,27 @@ export function checkUnsupportedConfig(
     );
   }
 }
+
+export function resolveConfigureWebpack(
```

Comment:
This function is a bit more complex than necessary. It shouldn't have a second parameter, the first if statement isn't necessary, and the nullish coalescing operator isn't necessary. If `configure-webpack.js` exists, then we want to handoff whatever is in that module to Vue CLI and let it validate the contents. The following should work similarly:

```typescript
export function resolveConfigureWebpack(projectRoot: string) {
  const configureWebpackPath = join(
    normalize(projectRoot),
    'configure-webpack.js'
  );
  const host = new virtualFs.SyncDelegateHost(new NodeJsSyncHost());

  return host.exists(configureWebpackPath)
    ? require(getSystemPath(configureWebpackPath))
    : undefined;
}
```